app/views/users/backup_code_setup/create.html.erb

```diff
@@ -14,21 +14,19 @@
 </p>
 
 <div class="margin-top-4 margin-bottom-5">
-  <table>
-    <tr>
-      <% [ @codes.first(@codes.length / 2), @codes.last(@codes.length / 2)].each do |section| %>
-        <td class="padding-0">
-          <% section.each do |code| %>
-            <code class="bold h2">
-              <%= RandomPhrase.format(code, separator: '-') %>
-            </code>
-            <br/>
-            <br/>
-          <% end %>
-        </td>
-      <% end %>
-    </tr>
-  </table>
+  <div class="grid-row">
```

Comment:
Minor: We could combine this with the `div` above, so we have

```html
<div class="grid-row margin-top-4 margin-bottom-5">
```

Reply:
Can't, doing so we are making the parent for "print","copy","download" to be a row too. It causes the buttons to be displayed differently.
![Screen Shot 2022-02-16 at 3 37 48 PM](https://user-images.githubusercontent.com/7727136/154353401-69a6f873-9675-4752-ae9e-29b8f39c23ce.png)